packages/admin-ui/src/pages/vpn/components/openvpn/OpenVpnDevicesHome.tsx

```diff
@@ -70,19 +74,26 @@ export default function DevicesHome() {
   }
 
   // Input errors
-
   const errors: string[] = [];
   if (input.length > maxIdLength)
     errors.push(`Device name must be shorter than {maxIdLength} characters`);
 
+  // Helper UI in case the OpenVpn client is bad
+  if (dnpsRequest.data) {
+    const vpnDnp = dnpsRequest.data.find(dnp => dnp.dnpName === vpnDnpName);
+    if (!vpnDnp) {
+      const url = `${installedRootPath}/${vpnDnpName}`;
+      return (
+        <Alert variant="secondary">
+          You must <NavLink to={url}>install the OpenVPN client</NavLink> to use
```

Comment:
It's not technically an OpenVPN client but an OpenVPN server. However it's not relevant here. You should just say "Install the OpenVPN package"